src/table.js

```diff
@@ -1322,6 +1322,72 @@ Table.prototype.sampleRowKeysStream = function(gaxOptions) {
   ]);
 };
 
+/**
+ * Returns true in callback if consitency check is true for
+ * generated consistency token.
+ * In-case consistency check returns false, retrial is done in interval
+ * of 5 seconds till 10 minutes, after that it returns false.
```

Comment:
Should this be user-configurable?

Reply:
That's a good idea.  We did not allow it in Java since we did exponential backoff type settings, but it seems like a useful feature with this implementation.